db/projects/eos/project.yaml

```diff
@@ -0,0 +1,66 @@
+name: EOS
+web: https://eos.io/
+start_date: "2017-06-26"
+assets:
+  - id: eos
+    type: Token
+    name: EOS
+    symbol: EOS
+    start_date: "2017-07-01"
+    token_properties:
+      platform: Ethereum
+      contract_address: "0x86fa049857e0209aa7d9e616f7eb3b3b78ecfdb0"
+      decimals: 18
+    ico_properties:
+      start_date: "2017-06-26"
+      end_date: "2018-06-01"
+      total_cap: 900000000
+      raised: 185000000 USD
+    networks:
+      - id: main
+        type: main
+        name: EOS Network
+        proof_type: DPoS
+        mineable: false
+        algorithm: DPoS
+        target_block_time: 120
+        maximum_block_size: 1000000
+        tools:
+          - type: explorer
+            url: https://etherscan.io/token/0x86fa049857e0209aa7d9e616f7eb3b3b78ecfdb0
+      - id: test
+        type: test
+        tools:
+          - type: explorer
+            url: http://www.eos.host/,
+    webids:
+      coinmarketcap: EOS
+      cryptocompare: eos
+  max_supply: 1000000000
+whitepapers:
+  - title: EOS.IO Technical White Paper v2
+team:
+  - name: Brendan Blumer
+    role: CEO
+    webids:
+      linkedin: https://www.linkedin.com/in/brendanblumer/
```

Comment:
`https://www.linkedin.com/in/brendanblumer/` neni webid, ale URL
musi tam byt `brendanblumer`